SocialFilter/sections/specific.txt

```diff
@@ -57,6 +57,8 @@ math-fun.net,polaris-official.com,apexheadline.com,aucfan.com,sist8.com,spaceshi
 !
+www.yna.co.kr##button.kkt
```

Comment:
Why do use `www` here and not on the next rule? Looks like without it is enough.